src/main/java/game/race/domain/support/Cars.java

```diff
@@ -1,50 +1,53 @@
-package game.race.support;
+package game.race.domain.support;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
-import game.race.Car;
-import game.race.support.move.MovePolicy;
+import game.race.domain.Car;
+import game.race.domain.support.move.MovePolicy;
 
 public class Cars {
 
-    private final List<Car> cars;
+    private final List<Car> carList;
 
-    public Cars() {
-        this.cars = new ArrayList<>();
+    public Cars(List<Car> carList) {
+        this.carList = carList;
     }
 
-    public List<Car> of(List<String> vehicleNames) {
-        cars.addAll(vehicleNames.stream()
-                                .map(Car::new)
-                                .collect(Collectors.toList()));
+    public static Cars of(List<String> vehicleNames) {
+        Cars cars = new Cars(new ArrayList<>());
+        cars.getCarList().addAll(vehicleNames.stream().map(Car::new).collect(Collectors.toList()));
 
         return cars;
```

Comment:
```suggestion
    public static Cars of(List<String> vehicleNames) {
        List<Car> cars = vehicleNames.stream().map(Car::new).collect(Collectors.toList()));
        return new Cars(cars);
```
객체를 생성하고 빈 리스트를 꺼내서 다시 넣는 것 보다
생성과 동시에 초기화 시키는 것이 좋을 것 같아요